src/alm/alm_cache.rs

```diff
@@ -1,3 +1,4 @@
+#![deny(missing_docs)] 
```

Comment:
no need to include this in all files; you just need this once in `src/lib.rs`

Reply:
okay, i'll be a wee bit more thorough this time